src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -1092,10 +1094,19 @@ public async Task MigrateDatabaseOffline()
             var overallProgress = result.Progress as SmugglerResult.SmugglerProgress;
             var operationId = ServerStore.Operations.GetNextOperationId();
             var processDone = new AsyncManualResetEvent(token.Token);
-
             // send new line to avoid issue with read key 
             process.StandardInput.WriteLine();
-            process.Exited += (sender, e) => { processDone.Set(); };
+            process.Exited += (sender, e) =>
```

Comment:
It would be better to attach to Exited before starting the process.

Reply:
Better is an understatement. This is required

Reply:
done